src/main/java/billgates/view/gui/TopMenuBar.java

```diff
@@ -114,4 +146,42 @@ public void loadFile(File file) {
         // CONTINUE!!!
         System.out.println("loaded");
     }
+
+    //Function to show a description of the function of this app
+    private void showFunctions(){
+        JOptionPane.showMessageDialog(null, "<html>Username, Password, and Sign In:" +
+                        "<br>You will need to enter a username and password to sign in. " +
+                        "If you have not signed in with this username <br>before, " +
+                        "you will be automatically registered and signed in when you click the \"Sign in\" button. " +
+                        "If you have <br>signed in with this username before and entered the correct password, " +
+                        "you will be successfully signed in <br>when you click the \"Sign in\" button." +
+                        "<br>Sign Out:" +
+                        "<br>When you click the \"Sign Out\" button, " +
+                        "you will be signed out from the account you just signed in to." +
+                        "<br>Back:" +
+                        "<br>Return from the splitter's bill to the main bill." +
+                        "<br>Add entry:" +
+                        "<br>Add a new entry to the bill. " +
+                        "You need to enter the information for this new entry from the pop-up window." +
+                        "<br>Delete entry:" +
+                        "<br>Delete the selected entry from the bill." +
+                        "<br>MenuBar -> File -> Import File:" +
+                        "<br>Select a csv file and add the information in it to the bill as new entries." +
+                        "<br>MenuBar -> Settings -> Appearance:" +
+                        "<br>Change the background color or font." +
+                        "<br>MenuBar -> Help -> How to use:" +
+                        "<br>Some description of how to use Bill Gates." +
+                        "<br>MenuBar -> Help -> Shortcut key:" +
+                        "<br>Some descriptions of how to use the shortcut keys.",
+                "How to use the Bill Gates", JOptionPane.PLAIN_MESSAGE);
+    }
+
+    //Function to show that how to use shortcut key
+    private void shortcut(){
+        JOptionPane.showMessageDialog(null, "<html>Shift + I: MenuBar -> File -> Import File"
+                        +"<br>Shift + A: MenuBar -> Settings -> Appearance"
+                        +"<br>Shift + H: MenuBar -> Help -> How to use"
+                        +"<br>Shift + S: MenuBar -> Help -> Shortcut key",
+                "Shortcut key", JOptionPane.PLAIN_MESSAGE);
```

Comment:
```suggestion
                "Shortcut keys", JOptionPane.PLAIN_MESSAGE);
```

Reply:
Also, this section can also utilize the 3 double quotes feature from Java 17, it will make your life a bit easier.